### PR TITLE
ISPN-6535 Added test listener to JCache integration tests

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-6535

### CODE REVIEW

File: integrationtests/cdi-jcache-it/src/test/java/org/infinispan/integrationtests/cdijcache/interceptor/CacheRemoveAllInterceptorTest.java [hunk 1]
```diff
@@ -49,6 +53,16 @@ public void beforeMethod() {
       assertTrue(customCache.isEmpty());
    }
 
+   @BeforeTest(alwaysRun = true)
+   protected void testClassStarted(ITestContext context) {
```
Comment (danberindei) [inline integrationtests/cdi-jcache-it/src/test/java/org/infinispan/integrationtests/cdijcache/interceptor/CacheRemoveAllInterceptorTest.java]: Should use `@Listeners(TestResourceTrackingListener.class)` instead, it's less noisy.

**Comment by danberindei** [comment]
@slaskawi The commit message is a bit too generic for my taste :)

**Comment by slaskawi** [comment]
Comments addressed

**Comment by tristantarrant** [comment]
Pushed to master, thanks

**Pull request closed without merging**